rebase.go

```diff
@@ -4,45 +4,58 @@ import (
 	"context"
 	"encoding/json"
 
-	"github.com/buildpack/imgutil"
 	"github.com/buildpack/lifecycle/metadata"
 	"github.com/pkg/errors"
 
-	"github.com/buildpack/pack/config"
+	"github.com/buildpack/pack/builder"
+
 	"github.com/buildpack/pack/style"
 )
 
 type RebaseOptions struct {
-	RepoName string
-	Publish  bool
-	SkipPull bool
-	RunImage string
+	RepoName          string
+	Publish           bool
+	SkipPull          bool
+	RunImage          string
+	AdditionalMirrors map[string][]string
```

Comment:
~Can we confirm that these are being passed in from the command?~

It seems like these aren't being passed from the `Rebase` command.

Reply:
We probably need to create or modify a test so that this doesn't happen frequently. 

We were discussion/thinking a unit test of cmd arguments to options would be sufficient and quick. There is also an argument to just modifying an acceptance test to do it.

Reply:
I modified a build and rebase acceptance test so we can have confidence we are wiring the mirrors all the way through.